src/storage/v3/vertex_accessor.hpp

```diff
@@ -146,12 +134,27 @@ class VertexAccessor final {
   bool operator!=(const VertexAccessor &other) const noexcept { return !(*this == other); }
 
  private:
+  /// Add a label and return `true` if insertion took place.
+  /// `false` is returned if the label already existed.
+  /// @throw std::bad_alloc
+  Result<bool> AddLabel(LabelId label);
+
+  /// Remove a label and return `true` if deletion took place.
+  /// `false` is returned if the vertex did not have a label already.
+  /// @throw std::bad_alloc
+  Result<bool> RemoveLabel(LabelId label);
+
+  /// Set a property value and return the old value.
+  /// @throw std::bad_alloc
+  Result<PropertyValue> SetProperty(PropertyId property, const PropertyValue &value);
+
   Vertex *vertex_;
   Transaction *transaction_;
   Indices *indices_;
   Constraints *constraints_;
   Config::Items config_;
   VertexValidator vertex_validator_;
+  const Schemas *schemas_;
```

Comment:
Where is this used? I cannot find any usage of this.

Reply:
Schemas will be removed